src/idea.js

```diff
@@ -0,0 +1,19 @@
+class Idea {
+  constructor(title, body){
+    this.id = Date.now();
```

Comment:
Something I'm just curious about - is there another way to create a unique identifier than Date.now()?